src/models/db/accounts.js

```diff
@@ -30,7 +30,11 @@ function * _upsertAccount (account, options) {
     existingAccount.setData(account)
     yield existingAccount.save(options)
   } else {
-    yield Account.create(account, options)
+    if (!account.balance) {
+      yield Account.create(_.merge({}, account, {balance: 0}), options)
+    } else {
+      yield Account.create(account, options)
+    }
```

Comment:
What code in five-bells-shared are you referring to? I thought this code in five-bells-ledger was what we were talking about: https://github.com/interledger/five-bells-ledger/blob/master/test/helpers/db.js#L39